webapp/src/Controller/API/JudgehostController.php

```diff
@@ -1339,7 +1339,7 @@ public function getJudgeTasksAction(Request $request): array
             // Note: we are joining on queue tasks here since if there is no more queue task, there is also no more
             // work to be done. If we would not do this join, the getJudgetasks would try to delete the queue task,
             // which is both slow and results in spamming the auditlog
-            ->innerJoin(QueueTask::class, 'qt', Join::WITH, 'qt.jobid = jt.jobid')
+            ->innerJoin(QueueTask::class, 'qt', Join::WITH, 'qt.judging = jt.jobid')
```

Comment:
So a `judgetask.jobid` could still refer to different types of tables?

Reply:
Yes, let's change that in a separate PR.